README.md

```diff
@@ -1 +1,157 @@
-# Metrics-webapp
\ No newline at end of file
+# Metrics webapp
+<!-- TABLE OF CONTENTS -->
+
+<details>
+  <summary>📗 Table of Contents</summary>
+  <ul>
+    <li><a href="#-Metrics-webapp">📖 Metrics webapp</a></li>
+    <li><a href="#-built-with-">🛠 Built With</a></li>
+    <li><a href="#-live-demo-">🚀 Live Demo </a></li>
+    <li><a href="#-getting-started-">💻 Getting Started</a></li>
+    <li><a href="#-author-">👥 Author </a></li>
+    <li><a href="#-future-features-">🔭 Future Features</a></li>
+    <li><a href="#-contributing-">🤝 Contributing</a></li>
+    <li><a href="#️-show-your-support-">⭐️ Show your support </a></li>
+    <li><a href="#-acknowledgments-">🙏 Acknowledgments </a></li>
+    <li><a href="#-license-">📝 License</a></li>
+  </ul>
+</details>
+
+<br>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics webapp <a name="about-project"></a>
+
+**Metrics webapp** is a website for chacking the air pollution in each city. It is a Single Page App (SPA) that allows users to:
+
+- Check the air pollution.
+- serch for the cites.
+
+</br>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JavaScript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://react-redux.js.org/">redux</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo](https://boo-shehab.github.io/Metrics-webapp/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ git
+ npm
+ node
+ Your favorite text editor
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/boo-shehab/Metrics-webapp.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Math Magicians
```

Comment:
- [ ] [REQUIRED] Let's change directory (cd) to `Metrics Webapp` and not `Math Magicians` so as not to mislead the users of this application.